### PR TITLE
populate animal details page

### PR DESCRIPTION
**Pull request by padomaitis**

### CODE REVIEW

File: src/pages/AnimalDetails.test.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+}));
+
+const mockedData = {
+    animal: {
+        id: 1,
+        name: 'Jupyteris',
+        image_url: 'https://picsum.photos/id/237/300/300',
+        details: {
+            birth_date: '1577836800000',
+            breed: 'Rodezijos ridžbekai',
```
Comment (rikius) [inline src/pages/AnimalDetails.test.tsx:32]: better to have English namings.

File: src/pages/AnimalDetails.tsx [hunk 2]
```diff
@@ -30,21 +40,46 @@ function AnimalDetails() {
...
+        return <p>Error!</p>;
+    }
+
+    if (!data?.animal) {
+        // TODO: replace with proper UI elements
+        return <p>No data!</p>;
+    }
+
+    const { animal } = data;
+    const birthDay = animal.details?.birthDate ? getAnimalAge(Number(animal.details.birthDate)) : '';
```
Comment (rikius) [inline src/pages/AnimalDetails.tsx:62]: Don't specify type here, better in Utility class `getAnimalAge`.

**Review by rikius — CHANGES_REQUESTED** [review]

File: src/pages/AnimalDetails.test.tsx (hunk 1 above)
Comment (padomaitis) [reply src/pages/AnimalDetails.test.tsx]: fixed

File: src/pages/AnimalDetails.tsx (hunk 2 above)
Comment (padomaitis) [reply src/pages/AnimalDetails.tsx]: fixed

**Review by rikius — APPROVED** [review]

**Pull request merged**